contents/docs/session-replay/mobile.mdx

```diff
@@ -11,134 +11,107 @@ Mobile Session Recording allows you to record user sessions on mobile devices. T
+    companion object {
+        const val POSTHOG_API_KEY = "<ph_project_api_key>"
+        // usually 'https://app.posthog.com' or 'https://eu.posthog.com'
+        const val POSTHOG_HOST = "<ph_instance_address>"
+    }
 
-        configuration.captureApplicationLifecycleEvents = true; // Record certain application events automatically!
-        configuration.captureScreenViews = true; // Capture screen views automatically!
-        
-        configuration.recording.screenRecordingEnabled = true
-        configuration.recording.logRecordingEnabled = true
-        configuration.recording.networkRecordingEnabled = true
-        configuration.recording.redactionMode = .automatic
+    override fun onCreate() {
+        super.onCreate()
 
-        return true
+        val config = PostHogAndroidConfig(
+            apiKey = POSTHOG_API_KEY,
+            host = POSTHOG_HOST // TIP: `host` is optional if you use https://app.posthog.com
+        )
+        config.sessionReplay = true
+        PostHogAndroid.setup(this, config)
     }
 }
 ```
 
+Add the OkHttp Interceptor.
 
-#### SwiftUI
-If you haven't already, crate the `AppDelegate.swift` file mentioned above and then in your `App` file add the following:
-
-```swift
-import SwiftUI
+```android_kotlin
+import com.posthog.PostHogOkHttpInterceptor
+import okhttp3.OkHttpClient
 
-@main
-struct AboutMeApp: App {
-    @UIApplicationDelegateAdaptor(AppDelegate.self) var appDelegate // Add this if not already there
-    
-    var body: some Scene {
-        WindowGroup {
-            ...
-        }
-    }
-}
+private val client = OkHttpClient.Builder()
+    .addInterceptor(PostHogOkHttpInterceptor(captureNetworkTelemetry = true))
+    .build()
 ```
 
-
 # Recording specific configuration
 
-```swift
-// The values shown are the defaults
-let config = PostHogRecorderConfig(
-    // Whether to enable recording. If false, all other options are ignored
-    screenRecordingEnabled: true,
-    // Whether to enable log recording. Logs with OSLog will be automatically recorded or you can log directly with PostHog.shared.recorder.log()
-    logRecordingEnabled: true,
-    // Records network requests via swizzling of NsUrlSession.
-    networkRecordingEnabled: true,
-    // What level of redcaction to perform
-    // -> .selective = only redact manually specified views with an accessibilityId matching what is specified in `redactionTags`.
-    // -> .automatic = redact all views with the name matching `redactionViews` and all classes matching `redactionClasses`
-    // -> .wireframe = renders a "wireframe" of the view, without any text or images
-    redactionMode: .selective,
-    redactionTags: ["ph-no-capture"],
-    redactionViews: PostHogRecorderConfig.DefaultRedactionViews,
-    redactionClasses: PostHogRecorderConfig.DefaultRedactionClasses,
-    // Hosts to ignore when recording network requests. By default the PostHog API host is ignored
-    ignoreNetworkHosts: []
-)
+The SDK configurations.
```

Comment:
Where does this go?

Reply:
PostHogAndroidConfig is during the SDK initialization, you will notice that is the same code snippet from the Installation step, so people will have context when they read the code snippet, also made more clear